### ISSUE TITLE
AttributeError: '_process_plot_var_args' object has no attribute 'prop_cycler'

### ISSUE THREAD

**Issue by Serendipity31** [awslabs/gluon-ts#3015]
## Description
The 'simple example' shown on the gluonTS home page plots the predictions from the DeepAREstimator (https://ts.gluon.ai/stable/index.html). It has three prediction windows, and shows the predictions for each window in a different color. When I try to run that code I get the error in the title of this post. 

If, instead of the code given in the example, I specify a color in forecast.plot(), the plot will show. Each forecast window simply uses shading based on whatever that color is.

## To Reproduce
This version (from the example given) produces the error:
```python
plt.plot(df["1954":], color="black")
for forecast in forecasts:
  forecast.plot()
plt.legend(["True values"], loc="upper left", fontsize="xx-large")
```
This version plots successfully, just with each window in shades of red.
```python
plt.plot(df["1954":], color="black")
for forecast in forecasts:
  forecast.plot(color="red")
plt.legend(["True values"], loc="upper left", fontsize="xx-large")
```

## Error message or code output

```python
Traceback (most recent call last):
  File "<string>", line 2, in <module>
  File "/Users/usr/.virtualenvs/deepAR_python_3_11_5/lib/python3.11/site-packages/gluonts/model/forecast.py", line 315, in plot
    color = maybe.unwrap_or_else(
            ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/usr/.virtualenvs/deepAR_python_3_11_5/lib/python3.11/site-packages/gluonts/maybe.py", line 179, in unwrap_or_else
    return factory()
           ^^^^^^^^^
  File "/Users/usr/.virtualenvs/deepAR_python_3_11_5/lib/python3.11/site-packages/gluonts/model/forecast.py", line 316, in <lambda>
    color, lambda: next(ax._get_lines.prop_cycler)["color"]
                        ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: '_process_plot_var_args' object has no attribute 'prop_cycler'
```

## Environment
- Operating system: Apple M1 Sonoma 14.0
- Python version:     3.11.5
- GluonTS version:   0.13.4
- Matplotlib versions: 3.7, 3.7.2, 3.8 (I get this error in each of these versions)

**Comment by lostella** [comment]
Hi @Serendipity31, I believe this was fixed in #3006, which is included in version 0.13.5

Could you try upgrading to see if the error disappears?

**Comment by Serendipity31** [comment]
Hi @lostella - Since the other issue I raised was fixed, I am now able to get to the point of generating the plot on my mac. I can confirm that the code given in the example does work without error. The plot it generates does not match the plot in the example, but that will be a separate issue, so I'll raise that separately in case anyone knows the answer as to why.
